autoload/lsp/omni.vim

```diff
@@ -83,6 +83,10 @@ function! s:handle_omnicompletion(server_name, startcol, complete_counter, data)
     endif
 endfunction
 
+function! lsp#omni#get_kind(match) abort
```

Comment:
can you rename this to `get_kind_text` or `get_text_from_kind`? That way we don't get confused with the actual LSP kind.

Reply:
also might want to change `s:kind_labels` to `s:kind_text_mappings`?